README.md

```diff
@@ -34,28 +34,40 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic html is a tagging system we use to organize our html to make it more readable to developers and computers alike. 
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+Display: inline will not allow you to adjust height and width, whereas display: block will. 
+
+Display: block also adds a break in the line, whereas display: inline does not and remains in the line. 
+
 3. What are the 4 areas of the box model?
 
+They are the main axis (x), the cross axis (y), the main/cross start and the main/cross end. 
```

Comment:
The four areas of the box model are the margin area, border area, padding area, and content area.